src/Microsoft.Dnx.Runtime/DependencyManagement/LockFilePackageLibrary.cs

```diff
@@ -17,5 +17,13 @@ public class LockFilePackageLibrary
         public string Sha512 { get; set; }
 
         public IList<string> Files { get; set; } = new List<string>();
+
+        public LockFilePackageLibrary() { }
+
+        public LockFilePackageLibrary(string name, SemanticVersion version)
```

Comment:
See below. Just trying to make the tests a little more readable. Didn't seem harmful